ebaysdk/utils.py

```diff
@@ -8,6 +8,7 @@
 '''
 import sys
 from lxml import etree as ET
+from xml.sax.saxutils import escape
```

Comment:
It is a standard library as far as I'm aware, yes. https://docs.python.org/2/library/xml.sax.utils.html